app/assets/stylesheets/application.css

```diff
@@ -400,11 +400,11 @@ body.modal-open{
 .modal{
   padding-right: 0 !important;
 }
-.error, .notice, .success {padding:.8em;margin-bottom:1em;border:2px solid #ddd;}
```

Comment:
Rails defines two types of flashes out of the box: notice and alert, and they have fancy accessors like `flash.alert` instead of `flash[:alert]`.

I've added styles for alert to be displayed in same way as error. Actually I'd propose to use `alert` everywhere instead of `error` to follow Rails defaults.

Reply:
thank you. I think these are remnants of ancient Rails habits (maybe 2.x?), when there used to be a flash[:error].